packages/trader/src/Modules/SmartChart/Components/Markers/delayed-accu-barriers-marker.tsx

```diff
@@ -7,7 +7,7 @@ type TDelayedAccuBarriersMarkerProps = {
     granularity?: number;
     is_dark_theme?: boolean;
     is_in_contract_details?: boolean;
-    marker_component: React.ComponentType<
+    marker_component?: React.ComponentType<
```

Comment:
<img width="528" alt="image" src="https://github.com/kate-deriv/deriv-app/assets/56330681/0a8e272f-cb93-48ec-bd56-dd339197addb">

marker_component prop doesn't have to be undefined as per the above code. Why are we making this undefined?

Reply:
Hi @balakrishna-deriv ! Previously `marker_component` (`accu_barriers_marker_component`) was not `undefined`, but after latest changes on master on line 258 a ternary was added: if it's `is_beta_chart`, then `accu_barriers_marker_component` is undefined :( 
So, I saw 2 options: 1) add `// @ts-expect error` for `<DelayedAccuBarriersMarker />` OR 2) make `marker_component` optional. Could you give some tips for this case please?
Does the `beta-chart` work properly for Accumulators (in Contract Details) as currently on production `marker_component` is `undefined`?

Reply:
> Does the beta-chart work properly for Accumulators (in Contract Details) as currently on production marker_component is undefined?

Yes, It works fine. We have moved the drawings to Smartcharts library.

Since we are using type assertion in the below line, we don't have to make `maker_component` optional prop. It should be fine without it. Let me know if there is a problem.
```
marker_component={
    accu_barriers_marker_component as React.ComponentProps<
                                typeof DelayedAccuBarriersMarker
                            >['marker_component']
 }
```

Reply:
Oops, you are absolutely right, sorry for my friezes